packages/KModal/KModal.vue

```diff
@@ -5,39 +5,43 @@
     class="k-modal"
```

Comment:
We should add an `aria-labelledby` and point it to the title

Reply:
Now using `aria-modal`, but `aria-labelledby` requires an `id`? Instead using `aria-label` with the `title` prop.... if they instead use the `header-content` slot with some sort of complex HTML, this won't work though, and wouldn't work for an `aria-labelledby` would it? Thinking of making `title` possibly a required prop?

Reply:
I have made the `title` prop required. It is always used for the `aria-label`. Added a new prop `hideTitle` if they don't want to display it for some reason.